src/main.rs

```diff
@@ -114,6 +114,18 @@ fn create_app<'a, 'b>(
                 .default_value(&default_memory)
                 .group("vm-config"),
         )
+        .arg(
+            Arg::with_name("memory-region")
+                .long("memory-region")
```

Comment:
This is almost nit-picking, but in order to not confuse it with the `vm-memory` crate memory region terminology, could we rename that one? I propose `--memory-zone` instead.

The user should not be confused with the `memory-region` terminology as hopefully this is transparent to users, but from a code perspective we end up talking about `memory_regions`, `ext_regions` and `custom_regions`. While the first ones actually mapped to `vm-memory` regions, the last one does not. It describes something that could be split into several `vm-memory` regions. That something could be called `memory_zones` instead.

Reply:
Sounds good!